internal/provider/resource_database.go

```diff
@@ -140,29 +144,33 @@ func resourceDatabaseCreate(ctx context.Context, d *schema.ResourceData, meta in
 	name := d.Get("name").(string)
 	keyspace := d.Get("keyspace").(string)
 	cloudProvider := d.Get("cloud_provider").(string)
-	region := d.Get("region").(string)
+	regions := d.Get("region").([] interface{})
```

Comment:
Q: I see where the regions are used later as expected type `string`,  any specific reason that we use interface{} type.  Are you thinking of a more complex structure for region in the future?

That would cleanup a lot of the conversion to string (e.g. `region.(string)`) being done in other areas of the code if we can go with `string` type.

Reply:
I did try that. But the type that is returned from the Terraform `ResourceData` (what `d` is here) is just a generic `interface{}`. So the only thing I could do to avoid the conversion to string is to create another variable and loop through the returned `interface{}`s and convert them. So it's not really saving anything.